### ISSUE TITLE
envinfo error in Edge

### ISSUE THREAD

**Issue by unadlib** [tabrindle/envinfo#184]
### Version

v7.7.2

### Env
Edge: 84.0.522.44
macOS: 10.13.6

### Error
```
(node:39622) UnhandledPromiseRejectionWarning: TypeError: (e || []).filter is not a function
    at /usr/local/lib/node_modules/reactant-cli/node_modules/envinfo/dist/envinfo.js:1:73314
    at async Promise.all (index 6)
(node:39622) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:39622) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by tabrindle** [comment]
closed via #185

**Comment by tabrindle** [comment]
fix in 7.7.3
